### DISCUSSION TITLE
Disable Platform UI Login Banner

### DISCUSSION THREAD

**Discussion by yodaphone** [inventree/InvenTree#7471] [Q&A]
Hi

Is it possible to disable the appearance of this?

Platform UI - the new UI for InvenTree - is ready to be tested.
Try it out now Here.

![image](https://github.com/inventree/InvenTree/assets/26359077/feb359eb-5db1-4f29-a27b-c3eff752fa10)

**Comment by matmair** [comment]
Please read/search the docs before asking - the option is listed there

**Comment by CommanderDer** [comment]
Hello, I also read and searched the documentation but didn't find anything either. 
Have you already found a solution? Thank you

**Reply by CommanderDer** [reply]
ok I found 
INVENTREE-CUSTOMIZE | customize.hide-pui-banner | PUI Banner | False
but how can  i change it?
`inventree config:set customize.hide-pui-banner=true ` dosen`t work
please help

**Reply by cabottech** [reply]
Add this to your config.yaml:

```yaml
customize:
   hide_pui_banner: true
```
